README.md

```diff
@@ -1,73 +1,203 @@
+      <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--light.svg">
+      <img alt="Supabase Logo" width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/logo-preview.jpg">
+    </picture>
+  </a>
 
-## Requirements
-**Python 3 higher**
+  <h1 align="center">Supabase Realtime Client</h1>
+
+  <h3 align="center">Send ephemeral messages with <b>Broadcast</b>, track and synchronize state with <b>Presence</b>, and listen to database changes with <b>Postgres Change Data Capture (CDC)</b>.</h3>
+
+  <p align="center">
+    <a href="https://supabase.com/docs/guides/realtime">Guides</a>
+    ·
+    <a href="https://supabase.com/docs/reference/python">Reference Docs</a>
+    ·
+    <a href="https://multiplayer.dev">Multiplayer Demo</a>
+  </p>
+</p>
+
+# Overview
+
+This client enables you to use the following Supabase Realtime's features:
+
+- **Broadcast**: send ephemeral messages from client to clients with minimal latency. Use cases include sharing cursor positions between users.
+- **Presence**: track and synchronize shared state across clients with the help of CRDTs. Use cases include tracking which users are currently viewing a specific webpage.
+- **Postgres Change Data Capture (CDC)**: listen for changes in your PostgreSQL database and send them to clients.
+
+# Usage
+
+## Installing the Package
 
-## Installation
 ```bash
-pip3 install realtime==1.0.2
+pip3 install realtime==2.0.0
 ```
 
-## Installation from source 
-```bash
-pip3 install -r requirements.txt
-python3 usage.py 
+## Creating a Channel
 
+```python
+import asyncio
+from typing import Optional
+from realtime.client import RealtimeClient
+from realtime.channel import RealtimeSubscribeStates
+
+client = RealtimeClient(REALTIME_URL, API_KEY)
+channel = client.channel('test-channel')
+
+def _on_subscribe(status: RealtimeSubscribeStates, err: Optional[Exception]):
+    if status == RealtimeSubscribeStates.SUBSCRIBED:
+        print('Connected!')
+    elif status == RealtimeSubscribeStates.CHANNEL_ERROR:
+        print(f'There was an error subscribing to channel: {err.message}')
+    elif status == RealtimeSubscribeStates.TIMED_OUT:
+        print('Realtime server did not respond in time.')
+    elif status == RealtimeSubscribeStates.CLOSED:
+        print('Realtime channel was unexpectedly closed.')
+
+await channel.subscribe(_on_subscribe)
+
+# Listen for all incoming events, often the last thing you want to do.
+await client.listen()
 ```
 
-## Quick Start 
+### Notes:
+
+- `REALTIME_URL` is `'ws://localhost:4000/socket'` when developing locally and `'wss://<project_ref>.supabase.co/realtime/v1'` when connecting to your Supabase project.
```

Comment:
/nit very minor point - would personally typically copy paste in browser so wondering  if we should consider removing the quotes so folks can directly copy paste

```suggestion
- `REALTIME_URL` is `ws://localhost:4000/socket` when developing locally and `wss://<project_ref>.supabase.co/realtime/v1` when connecting to your Supabase project.
```